extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -296,12 +296,12 @@ export class AmpIosAppBanner extends AbstractAppBanner {
    * @private
    */
   parseIosMetaContent_(metaContent) {
-    const parts = metaContent.replace(/\s/, '').split(',');
-    const config = {};
-    parts.forEach((part) => {
-      const keyValuePair = part.split('=');
-      config[keyValuePair[0]] = keyValuePair[1];
-    });
+    const config = Object.fromEntries(
```

Comment:
`Object.fromEntires` is extremely new (ES2019), and requires a polyfill to be used in relevant browsers. Please revert.

Reply:
Whoops, my bad

Reply:
Changed back to `reduce`, and extracted to a separate function for better readability